packages/slice-machine/lib/builders/SliceBuilder/SetupDrawer/index.tsx

```diff
@@ -91,18 +92,20 @@ const SetupDrawer: React.FunctionComponent<SetupDrawerProps> = ({
             )}
           </Flex>
         </Flex>
-        <HelpSection />
+        {!!storybook === false &&
+          (framework === Frameworks.next || framework === Frameworks.nuxt) && (
+            <StorybookSection framework={framework}></StorybookSection>
+          )}
       </Flex>
     </Drawer>
   );
 };
 
-const HelpSection = () => (
-  <Flex
-    sx={{
-      padding: "16px 20px",
-    }}
-  >
+const Section: React.FC<{
```

Comment:
Could it be possble to put the section components in a side file.
Just so we keep clean and small files :) for code readability I believe it is a good practice :)